exporters/etw/test/etw_provider_test.cc

```diff
@@ -0,0 +1,78 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifdef _WIN32
+
+#  include <gtest/gtest.h>
+#  include <string>
+
+/* TODO: this definition needs to be removed when TraceLoggingDynamic.h is OSS */
+#  ifndef HAVE_NO_TLD
+#    define HAVE_NO_TLD
+#  endif
+
+#  include "opentelemetry/exporters/etw/etw_provider_exporter.h"
+
+using namespace OPENTELEMETRY_NAMESPACE;
+
+TEST(ETWProvider, ProviderIsRegisteredSuccessfully)
+{
+  std::string providerName = "OpenTelemetry";
+  static ETWProvider etw;
+  auto handle = etw.open(providerName.c_str());
```

Comment:
I think you have to call `etw.close` for the handles you registered, to keep the state clean after tests? Otherwise those stay stored in the singleton `map` and my obfuscate results of other tests.